src/extension/rn-extension.ts

```diff
@@ -49,6 +50,10 @@ export function activate(context: vscode.ExtensionContext): void {
                     const nodeDebugPath = vscode.extensions.getExtension("andreweinand.node-debug").extensionPath;
                     const fsUtil = new FileSystem();
                     fsUtil.writeFile(path.resolve(__dirname, "../", "debugger", "nodeDebugLocation.json"), JSON.stringify({ nodeDebugPath })).done();
+                }).then(() => {
+                    let server = new ExtensionServer();
+                    server.setup();
```

Comment:
I'm rewriting how we handle and report the errors at the top level, so you can leave the code as-is and add a TODO, and I'll implement it as part of my fix/PR.